.vscode/settings.json

```diff
@@ -1,24 +1,28 @@
 {
-	"files.exclude": {
```

Comment:
I would not bother with the indentation, since this file will be modified by VS Code when changing the workspace settings.